AncientReligions/history/characters/irish.txt

```diff
@@ -23509,7 +23509,10 @@
 	culture="irish"
 	father = 7258
 	550.1.1={
-		birth="550.1.1" 
+		birth="550.1.1"
+		effect = {
+			set_global_flag = ancrel
```

Comment:
The various HIP modules use the name of the mod (emf, hip, viet, pb) as their "startup timer" and i'd like to be consistent with that.

I'll add a comment to that entry to document what it is used for.